en/components/general/ignite-ui-licensing.md

```diff
@@ -152,24 +152,44 @@ The following information is on how to setup authentication to our private npm r
 Update the azure-pipelines.yml with the following steps:
 
 ```cmd
-npm config set @infragistics:registry $(npmRegistry)
-```
+pool:
+  vmImage: 'Ubuntu 16.04'
 
-```cmd
-npm config set always-auth true --scope:@infragistics
-```
+steps:
+
+- script: echo "@infragistics:registry=$(npmRegistry)" >> ~/.npmrc
+  displayName: 'Npm add registry'
+
+- script: echo "$(igScope):always-auth=true" >> ~/.npmrc
+  displayName: 'Npm config'
+
+- script: echo "$(igScope):_auth=$(token)" >> ~/.npmrc
+  displayName: 'Npm config auth'
+
+- script: npm install
+  displayName: 'Install dependencies'
+  env:
+    AZURE_PIPELINES: "true"
+
+- script: npm run lint
+  displayName: 'Run lint'
+
+- script: npm run build
+  displayName: 'Generate live editing and build samples'
```

Comment:
Huh?
What are those extra stuff? Generate live editing, Lint and the `AZURE_PIPELINES` all seem out of scope